docs/source/kernel-vanilla.md

```diff
@@ -0,0 +1,19 @@
+## Vanilla
+
+This page describes the approach taken for integrating Enterprise Gateway into a vanilla set of hosts, without cluster resource managers.
+
+Following kernelspec is available on Vanilla mode:
+
++ python_distributed
+
+Install the `python_distributed` kernelspec on all nodes.
+
+``` Bash
+wget https://github.com/jupyter/enterprise_gateway/releases/download/v2.0.0rc1/jupyter_enterprise_gateway_kernelspecs-2.0.0rc1.tar.gz
+KERNELS_FOLDER=/usr/local/share/jupyter/kernels
+tar -zxvf jupyter_enterprise_gateway_kernelspecs-2.0.0.dev2.tar.gz --strip 1 --directory $KERNELS_FOLDER/python_distributed/ python_distributed/
+```
+
+The `python_distributed` kernelspec uses `DistributedProcessProxy` which is responsible for the launch and management of kernels distributed across and explicitly defined set of hosts using ssh. Hosts are determined via a round-robin algorithm (that we should make pluggable someday).
+
+**QUESTIONS: how to set the list of hosts - Also, SSH should be available passwordless, right? Which user should be used?***
```

Comment:
The set of remote hosts are derived from two places.  
1. The configuration option: `EnterpriseGatewayApp.remote_hosts` who's default value comes from the env variable `EG_REMOTE_HOSTS` - which, itself, defaults to 'localhost'.
2. The config option can be [overridden on a per-kernel basis](https://jupyter-enterprise-gateway.readthedocs.io/en/v2.0.0rc1/config-options.html#per-kernel-configuration-overrides) if the `process_proxy` stanza contains a `config` stanza where there's a `remote_hosts` entry.  If present, this value will be used instead.

Unfortunately, it doesn't look like the SSH requirement is very well stated.  We touch on this when describing [`DistributedProcessProxy`](https://jupyter-enterprise-gateway.readthedocs.io/en/v2.0.0rc1/system-architecture.html#distributedprocessproxy) and in the [troubleshooting section](https://jupyter-enterprise-gateway.readthedocs.io/en/v2.0.0rc1/troubleshooting.html), but nothing of any substance.

cc: @lresende 

We recommend running EG as the _gateway user_ which is touched on in the security section - but mostly relative to impersonation topics.